timemachine/fe/rbfe.py

```diff
@@ -617,12 +636,7 @@ def estimate_relative_free_energy(
     single_topology = SingleTopology(mol_a, mol_b, core, ff)
 
     if lambda_schedule is None:
-        lambda_schedule = np.array(
-            [0.0, 0.02, 0.04, 0.06, 0.07, 0.08, 0.11, 0.13, 0.15, 0.17, 0.18, 0.20, 0.25, 0.32, 0.42]
-        )
-        lambda_schedule = np.concatenate([lambda_schedule, (1 - lambda_schedule[::-1])])
-        if n_windows:
-            lambda_schedule = interpolate_pre_optimized_protocol(lambda_schedule, n_windows)
+        lambda_schedule = construct_pre_optimized_relative_lambda_schedule(n_windows)
```

Comment:
Isn't the default here that n_windows is None? Don't see anything that protects against this.

Reply:
good catch

Reply:
fixed in [866b5fb](https://github.com/proteneer/timemachine/pull/894/commits/866b5fbc0f1316fef016ece171a3a797afef1a70)